### PR TITLE
Public Url for Paths

### PR DESCRIPTION
**Pull request by HeapUnderfl0w**
This PR augments microbin with the `--public-path` option.
This allows setting a public "root" path enabling the use behind a reverse proxy / in a subpath.

### CODE REVIEW

**Comment by szabodanika** [comment]
Hi @HeapUnderfl0w, cheers for this PR, it seems very useful. Would you mind documenting this feature in README.MD as well (with an example as well perhaps)?

**Comment by HeapUnderfl0w** [comment]
Shure!. Will get to that later today.

File: README.MD [hunk 1]
```diff
@@ -330,6 +330,64 @@ Displays your MicroBin's version information.
...
+    # Your normal http root
+    root * /var/www/html
+    file_server
+    
+    # Route all requests to past
+    handle_path /paste/* {
+        reverse_proxy http://127.0.0.1:31333
+    }
+}
+```
```
Comment (HeapUnderfl0w) [inline README.MD:389]: @szabodanika let me know if this is too much for the options list.

I feel like a full example is needed to illustrate what the option should be used for.

Comment (szabodanika) [reply README.MD:389]: I think this is great, if the options list (and the readme file) become too long and hard to navigate I will set up a separate wiki/documentation page at one point. Thank you!

**Pull request merged**